docs/processes/quality-assurance/templates/test-strategy-template.md

```diff
@@ -0,0 +1,54 @@
+## **Test Strategy (Sample)**
+
+**Introduction**
```

Comment:
**suggestion (non-blocking):** Could we use heading semantics for all below sections. 👇🏼 

```md
# Test Strategy

## Introduction

### Examples

## References

### Examples

...
```

Reply:
I like this suggestion. Thanks @tancnle  

Incorporating 👍